<launch>
  <include file="$(find bebop_driver)/launch/bebop_node.launch"/>
  <!-- Not possible to set the external pose with the bebop -->
  <node pkg="drone_arena" type="bebop_controller.py" name="fence_controller" output="screen" ns="bebop">
    <remap from="cf_odom" to="odom"/>
    <param name="enable_fence" value="true"/>
    <param name="frame_id" value="odom"/>
    <param name="publish_target" value="False" />
    <param name="publish_body_vel" value="False" />
    <param name="publish_cmd" value="True" />
    <param name="teleop_mode" value="1"/>
    <param name="max_speed" value="1.0"/>
    <param name="max_vertical_speed" value="1.0"/>
    <param name="max_angular_speed" value="2.0"/>
    <param name="tau" value="0.5"/>
    <param name="eta" value="1.0"/>
    <param name="delay" value="0"/>
    <param name="joy_set_teleop_mode" value="true"/>
    <rosparam param="pos_bounds">[[-inf, inf], [-inf, inf], [0.2, 2.0]]</rosparam>
  </node>
  <include file="$(find drone_arena)/launch/joy_teleop.launch">
    <arg name="teleop_config" default="$(find drone_arena)/config/log710-demo-generic.yaml" />
    <arg name="namespace" value="bebop"/>
  </include>
</launch>
